src/preattestation.rs

```diff
@@ -350,9 +356,9 @@ mod idblock {
 
         // Formatted in Base-64 since it's the format QEMU takes.
         let id_block_string =
-            general_purpose::STANDARD.encode(bincode::serialize(&measurements.id_block)?);
+            general_purpose::STANDARD.encode(measurements.id_block.to_bytes()?.as_ref());
         let id_auth_string =
-            general_purpose::STANDARD.encode(bincode::serialize(&measurements.id_auth)?);
+            general_purpose::STANDARD.encode(measurements.id_auth.to_bytes()?.as_ref());
```

Comment:
```suggestion
            general_purpose::STANDARD.encode(&measurements.id_auth.to_bytes()?);

```

Reply:
I'm getting a compiler warning saying we don't even need to reference this:
```suggestion
            general_purpose::STANDARD.encode(measurements.id_auth.to_bytes()?);
```